CMakeLists.txt

```diff
@@ -58,8 +58,13 @@ endif()
 if(EMSCRIPTEN)
   message("Building for Emscripten")
   set(MANIFOLD_FLAGS -fexceptions -D_LIBCUDACXX_HAS_THREAD_API_EXTERNAL -D_LIBCUDACXX_HAS_THREAD_API_CUDA)
-  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -sALLOW_MEMORY_GROWTH=1 -fexceptions -sDISABLE_EXCEPTION_CATCHING=0")
+  # ALLOW_MEMORY_GROWTH is slow, so we just allow 2gb of memory usage
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -sMODULARIZE=1 -sEXPORT_ES6=1 -sINITIAL_MEMORY=2gb -fexceptions -sDISABLE_EXCEPTION_CATCHING=0")
   set(MANIFOLD_PYBIND OFF)
+  if(MANIFOLD_PAR STREQUAL "TBB")
+    set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} -pthread)
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pthread -sPTHREAD_POOL_SIZE=\"(typeof window == 'undefined')?4:(()=>navigator.hardwareConcurrency)()\"")
```

Comment:
If we don't set it ourselves, `PTHREAD_POOL_SIZE` defaults to zero and I think it is a link-time constant, so we must set it to some sensible value here...